engines/neverhood/module.mk

```diff
@@ -4,16 +4,16 @@ MODULE_OBJS = \
 	background.o \
 	blbarchive.o \
 	console.o \
-	detection.o \
 	diskplayerscene.o \
 	entity.o \
 	gamemodule.o \
 	gamevars.o \
 	graphics.o \
 	klaymen.o \
 	menumodule.o \
+	metaengine.o \
 	microtiles.o \
-	module.o \
+	module_scene.o \
```

Comment:
Why this rename?

Reply:
My commit notes for this say:
- The name was clashing with one of the audio files.
- New changes were spread to the detection files, so no idea how this file is involved.
- In the meanwhile, this commit should fix a build error, if any.

VStudio was having a problem with this file and one in the main scummvm subproject because they had the same name. I know we have the tool that checks for clashing names and renames it acc in dev tools, but this was still an issue and was the cause of compilation error.